client/shipping-label/components/label-purchase-modal/packages-step/add-item.js

```diff
@@ -67,24 +68,22 @@ const AddItemDialog = ( {
 				isFullScreen={ false }
 				onClickOutside={ closeAddItem }
 				onClose={ closeAddItem }
-				additionalClassNames="wcc-root wcc-label-packages-dialog" >
-			<div className="wcc-label-packages-dialog__content">
-				<h1 className="form-section-heading">{ __( 'Add item' ) }</h1>
-				<div className="wcc-label-packages-dialog__body">
-					<p>
-						{ __( 'Which item would you like to add to {{pckg/}}?', {
-							components: {
-								pckg: getPackageNameElement( openedPackageId ),
-							}
-						} ) }
-					</p>
-					{ itemOptions }
-				</div>
-				<ActionButtons buttons={ [
-					{ label: __( 'Add' ), isPrimary: true, onClick: () => moveItem( sourcePackageId, movedItemIndex, openedPackageId ) },
-					{ label: __( 'Close' ), onClick: closeAddItem },
-				] } />
+				additionalClassNames="wcc-root packages-step__dialog" >
```

Comment:
ESLint only checks for `className` props. This is `additionalClassNames`. I'm not gonna complain, we *need* the `wcc-root` class here, it's the root class for *all* our styles.